autorest/codegen/models/lro_operation.py

```diff
@@ -44,6 +44,9 @@ def __init__(
         self.lro_options = yaml_data.get("extensions", {}).get("x-ms-long-running-operation-options", {})
 
     def set_lro_response_type(self) -> None:
+        if not self.responses:
+            self.lro_response = None
```

Comment:
Unecessary assignment (done in init already)

Reply:
got rid of assignment